src/solver/svd/svd.hpp

```diff
@@ -12,7 +12,7 @@ namespace neon
 class svd
 {
 public:
-    svd();
+    svd(){}; // No idea why this is here
```

Comment:
If you want the default constructor, you can use `svd() = default;`.  Also the semi-colon after the `{};`.